bundle/src/bundle_execution.rs

```diff
@@ -264,6 +265,16 @@ pub fn load_and_execute_bundle<'a>(
                 account_overrides.upsert_account_overrides(
                     bank.get_account_overrides_for_simulation(&account_keys),
                 );
+
+                // An unfrozen bank's state is always changing. Need to take a snapshot of accounts to avoid race conditions.
```

Comment:
can you add comment - this doesn't perfeclty prevent race conditions but it does minimize them